examples/examples_dotnet_test.go

```diff
@@ -20,7 +25,31 @@ func TestSimpleCs(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestRoleCs(t *testing.T) {
+	test := getCsharpBaseOptions(t).
+		With(integration.ProgramTestOptions{
+			Dir:         filepath.Join(getCwd(t), "role-cs"),
+			SkipRefresh: true,
```

Comment:
I assume this is because refreshing turns the string policy into an object? Maybe worth a comment here about the behavior

Reply:
This was not intentional, it was copied from the test above. I'll remove all of these as this PR should fix all refreshes 🤞